index.html

```diff
@@ -72,5 +88,177 @@
 	  	
 	</div>
 </section>
+
+<section id="body">
+	<div class="grid-body">
+		<h1 class="recent-work">
+			My Recent Works
+		</h1>
+		<div class="stories">
```

Comment:
-  _[optional]_ Please I strongly recommend changing this and the others to `h2`, `h3`, or any other `hx`. As part of the best practices, its recommended to have only one `h1` tag in your file